carrent/carrentapp/static/carrentapp/styles/layout.css

```diff
@@ -0,0 +1,13 @@
+#header
+{
+  height: 80px;
+  text-align: center;
+  padding-top:20px;
+  padding-bottom:20px;
```

Comment:
Przy zmianę rozmiaru okna przeglądarki przyciski zamiast rozszerzać diva w którym się znajduja to nachodzą na innego diva